src/main.rs

```diff
@@ -281,7 +281,10 @@ fn main() {
     }
 
     // Specify target OS
-    let platform: PlatformType = args.platform.unwrap_or_else(PlatformType::current);
+    let platforms: Vec<PlatformType> = match args.platform {
+        Some(ref p) => p.clone(),
+        None => vec![PlatformType::current()],
+    };
```

Comment:
Hm, this seems wrong. We don't need ownership of the vector, a slice should totally suffice. I guess we are running into borrowing issues later, but we should fix those instead.

Depending on how difficult this turns out to be, we can also postpone a refactor, but I would not want this kind of fixing to pile up